libs/application/templates/university/src/fields/Review/SchoolCareerReview.tsx

```
@@ -45,9 +45,31 @@ export const SchoolCareerReview: FC<Props> = ({
               )}: ${educationItemExemption.moreDetails}`}</Text>
-              <Text>{`${formatMessage(review.labels.documents)}: TODO`}</Text>
+              {educationItemExemption.degreeAttachments && (
+                <Text>{`${formatMessage(review.labels.documents)}:`}</Text>
+              )}
+              {educationItemExemption.degreeAttachments?.map((file) => {
+                return (
+                  <Box
+                    display="flex"
+                    alignItems="center"
+                    marginBottom="smallGutter"
+                  >
```

Biome

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)